### PR TITLE
remove user script from script output in sys.upgrade_log.mdx

### PR DESCRIPTION
**Pull request by kbinreallife**
### Problem

accidentally left my script name in script output

### CODE REVIEW

File: docs/scripting/trust_scripts/sys.upgrade_log.mdx [hunk 1]
```diff
@@ -57,7 +57,6 @@ Get a range with count:<count> and start:<offset>
 #### Script
 
 ```
->>specs
```
Comment (seanmakesgames) [inline docs/scripting/trust_scripts/sys.upgrade_log.mdx:60]: This diff does not line up with the PR title

**Review by seanmakesgames — COMMENTED** [review]
maybe you missed making the right change?

File: docs/scripting/trust_scripts/sys.upgrade_log.mdx (hunk 1 above)
Comment (matr1x-hackmud) [reply docs/scripting/trust_scripts/sys.upgrade_log.mdx:60]: The context is kb was using `kbeeb.specs` to get the subscript output, and accidentally left the CLI call to `kbeeb.specs` in the article. I'm happy for this to get removed in-line with the PR title

Comment (seanmakesgames) [reply docs/scripting/trust_scripts/sys.upgrade_log.mdx:60]: oh haha. I get it.

**Review by seanmakesgames — APPROVED** [review]

**Pull request merged**